h2o-core/src/main/java/water/util/Log.java

```diff
@@ -81,7 +81,7 @@ public static void httpd( String method, String uri, int status, long deltaMilli
     l.info(s);
   }
 
-  public static void info( String s, boolean stdout ) { if( _level >= INFO ) write0(INFO, stdout, s); }
+  public static void info( String s, boolean stdout ) { if( _level >= INFO ) write0(INFO, stdout, new String[]{s}); }
```

Comment:
Sorry, I chose a bad explanation. It initialises the logger but does not handle the buffered messages. All logging should go through this method `private static void write0( int lvl, boolean stdout, Object objs[] ) {` as it checks buffered messages and internally calls the `write0( int lvl, boolean stdout, String s )` which initialises the logging. The `Log.info` skipped the first one.